Documentation/Blazorise.Docs/Pages/Docs/Components/Tables/TablePage.razor

```diff
@@ -130,6 +130,21 @@
     <DocsPageSectionSource Code="TableDarkHeaderExample" />
 </DocsPageSection>
 
+<DocsPageSection>
```

Comment:
It might be good to implement desktop/mobile mode in the examples so that user can switch the view. This can be done by using the `iframe` and setting the width. 

Check how it is done on Blocks https://github.com/Megabit/BlazoriseSupport/blob/master/WebApps/BlazoriseSupport.Blocks/Components/BlockPageSectionContent.razor